pytests/noxfile.py

```diff
@@ -14,6 +14,11 @@ def test(session: nox.Session):
     except CommandFailed:
         # No binary wheel for numpy available on this platform
         pass
+    try:
+        session.install("--only-binary=gevent", "gevent>=22.10.2")
+    except CommandFailed:
+        # No binary wheel for numpy available on this platform
```

Comment:
numpy?

Reply:
Ah that's a copy and paste from above, good catch 👍